targets/TARGET_NORDIC/TARGET_NRF5_SDK13/spi_api.c

```diff
@@ -0,0 +1,537 @@
+/* 
+ * Copyright (c) 2013 Nordic Semiconductor ASA
```

Comment:
2017 in the new fileS or 2016 as I noticed some files were commited in that year?